### DISCUSSION TITLE
Support switching language

### DISCUSSION THREAD

**Discussion by NookLook2014** [Data-Centric-AI-Community/fg-data-profiling#957] [Polls]
This is a great tool, and I'm wandering whether there is language mapping setting to support display different language in the html report? Or shall i modify the source code to achieve that?

**Comment by sbrugman** [comment]
The report and interface are currently in English, and switching languages is at this time is not a feature on the roadmap. However, if we find more people are in need of this feature, then we can consider adding it to the roadmap (please upvote the top post of this issue in that case!).

Regarding the implementation, this depends. For an ad-hoc translation, you could modify the HTML templates, rendering components and possibly other places where text is included. If you are interested in contributing support for multiple languages, then a mapping is preferred (e.g. via [gettext](https://docs.python.org/3/library/gettext.html))
